src/store/action-types.js

```diff
@@ -11,3 +11,4 @@ export const PEER_AUTHENTICATE = 'peerAuthenticate'
 export const REMOVE_REMOTE_PEER_ID = 'removeRemotePeerId'
 export const CHANGE_REMOTE_PEER_ID = 'newRemotePeerId'
 export const HANDLE_PEER_CONNECTION_ERROR = 'handlePeerConnectionError'
+export const FETCH_EDGE_DEVICE_DETAILS = 'FETCH_EDGE_DEVICE_DETAILS'
```

Comment:
@vickywane please follow the naming convention for other actions. The string value corresponds to a JS function name so it should be camel cased.

Reply:
@vickywane did you see this comment?

Reply:
Yes. 

I wasn't done with all changes at the time i made the last comment on this pull request.